cmd/analyze.go

```diff
@@ -434,71 +435,125 @@ func (c *coremarkAnalyzer) Close() (err error) {
 	return nil
 }
 
-const netCSVHeader = "Cloud,Date,MachineType,Throughput,ThroughputUnits,minLat," +
-	"meanLat,p90Lat,p99Lat,maxLat,latStdDev,txnRate"
+const netCSVHeader = "testMode,Cloud,DateTime(Zulu),MachineType,DiskType,ClientRegion,ServerRegion," +
+	"MinThrpt,MeanThrpt,MaxThrpt,ThrptUnit,ExpectedThrpt,#Streams," +
+	"RecvBufferSize(bytes),SendBufferSize(bytes),ThrptTestDuration(seconds),LatTestDuration(seconds)," +
+	"minLat(microseconds),meanLat(microseconds),p90Lat(microseconds),p99Lat(microseconds),maxLat(microseconds)," +
+	"LastStdDev,TxnRate,ThrptTimeSeriesPlotPath"
 
 type networkResult struct {
-	throughput      float64
-	throughputUnits string
-	minLat          float64
-	meanLat         float64
-	p90Lat          float64
-	p99Lat          float64
-	maxLat          float64
-	latStdDev       float64
-	txnRate         float64
-	modtime         time.Time
+	// testMode can be either "cross-region" or "intra-az".
+	testMode     string
+	diskType     string
+	machineType  string
+	clientRegion string
+	serverRegion string
+
+	// Latency results.
+	latTestDuration   string
+	minLatMicrosec    string
```

Comment:
Nit: I haven't checked if we're using a specific convention for time units in crdb, but this convention is fairly prevalent, and one that I typically use,

```
// minimum latency in microseconds
minLatencyMicros
// minimum latency in milliseconds
minLatencyMs
// minimum latency in seconds
minLatencySeconds
// minimum latency in nanoseconds
minLatencyNanos
```

Reply:
Done changing them to `xxxLatencyMicros`.